source/deployment/cluster.rst

```diff
@@ -420,6 +420,15 @@ Does Mattermost support multi-region High Availability deployment?
 
 Yes. Although not officially tested, you can set up a cluster across AWS regions, for example, and it should work without issues.
 
+What does Mattermost recommend for diaster recovery of the databases?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When deploying Mattermost in a High Availability configuration, we recommend using a database load balancer between Mattermost and your database. Depending on your deployment this needs more or less consideration.
```

Comment:
Do we mean depending on the size of the deployment, your choice of load balancer will differ?

Reply:
@justinegeffen Do you think removing this sentence would make it hard to transition to the two following statements? I was trying to come up with a better transition but was finding it challenging.

Reply:
I like the intention of the statement, and thinking something like this could work:

"The load balancer configuration depends on the scale of your deployment, so requirements will differ."